services/drupal/web/modules/custom/epa_core/epa_core.install

```diff
@@ -93,3 +93,18 @@ function _epa_core_create_news_release_type_term() {
   $term->field_term_days_til_review = 730;
   $term->save();
 }
+
+/**
+ * As part of https://forumone.atlassian.net/browse/EPAD8-1962 to set cardinality
+ * of the Press Office field to 1 we will remove records where delta > 0.
+ */
```

Comment:
Let's be a bit more defensive here and guard against cases where there's a value here that's stored in a delta higher than 0, while not having a 0-delta value.